yarn.lock

```diff
@@ -8222,6 +8222,14 @@ matrix-widget-api@^0.1.0-beta.18:
     "@types/events" "^3.0.0"
     events "^3.2.0"
 
+matrix-widget-api@^1.0.0:
```

Comment:
The lockfile also needs to remove the beta.18 version, otherwise we can end up with two widget-apis in production.

Reply:
(to do this, update the commit for the resolved version of the react-sdk to your merged react-sdk PR, then `yarn install`. This should cause it to pull the latest definitions and update automatically)

Reply:
Thanks, I had to fiddle with both package.json and the lock file and `yarn install --force` for a bit, but it seems to have worked.